tox.ini

```diff
@@ -3,7 +3,7 @@ envlist =
     flake8
     isort
     py{35,36,37,38}-dj{22}
```

Comment:
Could you please drop python 3.5 from here and add 3.9 to both django 2.2 and 3.x?